doc/main/tbb_userguide/Task_Scheduler_Bypass.rst

```diff
@@ -0,0 +1,20 @@
+.. _Task_Scheduler_Bypass:
+
+Task Scheduler Bypass
+=====================
+
+Scheduler bypass is an optimization where you directly specify the next task to run. 
+According to the rules of execution in  :doc:`How Task Scheduling Works <How_Does_Task_Scheduler_Works>`, 
+spawning of the new task to be executed by the current thread, involves these steps:
+
+ -  Push a new task onto the thread's deque.
+ -  Continue to execute the current task until it is completed.
+ -  Take a task from the thread's deque, unless it is stolen by another thread.
+
+Steps 1 and 3 introduce unnecessary deque operations or, even worse, allow stealing that can hurt 
+locality without adding significant parallelism. These problems can be avoided by returning next task to execute 
+instead of spawning it. When using the method shown in :doc:`How Task Scheduling Works <How_Does_Task_Scheduler_Works>`,
+the returned task becomes the next task executed by the thread. Furthermore, this approach almost guarantees that 
```

Comment:
The task does not always become the the next task due to Flow Graph priorities. Maybe `the returned task` is considered first to be executed by the thread`?

Reply:
good point, thanks for idea

Reply:
chnged